### PR TITLE
[TESTNET-69] create celery task for btc-usd and add multiple  dydx asset support 

### PR DESCRIPTION
**Pull request by Akhilleshgoswami**

### CODE REVIEW

File: components/dydx_order_manager.py [hunk 1]
```diff
@@ -277,6 +278,112 @@ def withdraw_from_dydx(self, recipient_address, dydx_client):
...
             raise Exception(e)
 
+    #    open order on dydx celery task algo
+    #     asset type -- ETH-USD,BTC-USD etc
+    def open_order_on_dydx(self, order_details):
+        dydx_asset_instance = asset_dydx_instance[order_details["asset_pair"]]
+        dydx_order_manager = DydxOrderManager(dydx_asset_instance)
+        asset_open_position = dydx_order_manager.position_status(
+            "position_status", order_details["binance_asset_pair"]
+        )
```
Comment (Prithviraj8) [inline components/dydx_order_manager.py]: what does this give you ?

File: components/dydx_order_manager.py [hunk 2]
```diff
@@ -277,6 +278,112 @@ def withdraw_from_dydx(self, recipient_address, dydx_client):
...
+        data = dydx_order_manager.get_asset_price_and_size(
+            order_details["asset_oracle_address"], dydx_asset_instance
+        )
+        asset_market_price = data["price"]
+
+        asset_position_size = data["size"]
+
+        if order_details["asset_trigger_price"] is None:
+            print("Start::calculate open close")
```
Comment (Prithviraj8) [inline components/dydx_order_manager.py]: is this print statement needed ? 
If yes, structure the print statement as such:
For celery tasks, "Start :: <name of the task>"
For any other file, "ClassName :: <print statement>"

Comment (Prithviraj8) [reply components/dydx_order_manager.py]: Do make the necessary changes for every print statement

File: components/dydx_order_manager.py (hunk 1 above)
Comment (Akhilleshgoswami) [reply components/dydx_order_manager.py]: it return the  true if the  asset position is open on dydx else false

File: components/dydx_order_manager.py (hunk 2 above)
Comment (Akhilleshgoswami) [reply components/dydx_order_manager.py]: sure

File: components/dydx_order_manager.py (hunk 1 above)
Comment (Prithviraj8) [reply components/dydx_order_manager.py]: oh then can you rename it to `is_positiion_open`

File: components/dydx_order_manager.py (hunk 2 above)
Comment (Prithviraj8) [reply components/dydx_order_manager.py]: oh I think there are some print statements inside dydx_order_manager functions without the above convention

File: components/dydx_order_manager.py [hunk 3]
```diff
@@ -277,6 +278,110 @@ def withdraw_from_dydx(self, recipient_address, dydx_client):
...
+            else:
+                print(
+                    f"class::DydxOrderManager  { order_details['asset_pair']} {asset_trigger_price} is less than market price {asset_market_price}"
+                )
+        else:
+            if asset_trigger_price >= asset_market_price:
+                print(f"{order_details['asset_pair']} position is already open")
+            else:
+                print(
+                    f"class::DydxOrderManager {order_details['asset_pair']} trigger price {asset_trigger_price} is less than market price {asset_market_price}"
```
Comment (Prithviraj8) [inline components/dydx_order_manager.py]: `print(DydxOrderManager :: f"{order_details['asset_pair']} position is already open")`

**Review by Prithviraj8 — APPROVED** [review]

**Pull request merged**